sdk/cosmos/azure-cosmos/azure/cosmos/_global_endpoint_manager.py

```diff
@@ -137,8 +137,7 @@ def _GetDatabaseAccount(self, **kwargs):
                     return database_account
                 except exceptions.CosmosHttpResponseError:
                     pass
-
-            return None
+            raise
```

Comment:
Isn't this a breaking change? Earlier this used to return and now it will raise error, breaking customer's current code. 
Behavior breaking change - so either we should document it properly in the changelog, or think how we can avoid this. 
Worth getting some guidance from Central team on this.

Reply:
It is and it isn't - users would still run into an error, just not the authentication error it is now.

Reply:
I agree that we shouldn't have been blindly swallowing errors here - the default should be to raise.
Though it could be worth investigating if this was originally written to target suppressing specific errors, or if there are scenarios where it makes sense to have the value as None in the case of some failures.
Definitely an auth error should be raised.

Reply:
I was trying to think of a scenario that would benefit from this behavior, but I couldn't find any to be honest - skipping the fetching of the user's database account information is bound to cause errors in the SDK's behavior sooner or later, so skipping on that step is something we should not have allowed at any point.